content/_data/roadmap/roadmap.yml

```diff
@@ -40,7 +40,7 @@ categories:
       status: future
     - name: Pipeline as YAML
       description: Official support for defining Jenkins Pipelines in YAML, without additional Pipeline libraries
-      status: near-term
+      status: current
       link: https://jenkins.io/projects/gsoc/2020/project-ideas/pipeline-as-yaml-experiment/
```

Comment:
is this happening as part of gsoc or is it a plugin that someone else is writing?

Reply:
It will be part of the pipeline-authoring SIG roadmap. Someone else is writing